rust/operator-binary/src/security/oidc.rs

```diff
@@ -25,15 +31,21 @@ pub enum Error {
     },
 
     #[snafu(display(
-        "found existing secret [{}/{}], but key {} is missing",
-        name,
-        namespace,
-        STACKABLE_OIDC_ADMIN_PASSWORD_KEY
+        "found existing admin password secret {secret:?}, but the key {STACKABLE_OIDC_ADMIN_PASSWORD_KEY} is missing",
     ))]
-    OidcAdminPasswordKeyMissing { name: String, namespace: String },
+    OidcAdminPasswordKeyMissing { secret: ObjectRef<Secret> },
```

Comment:
note: Technically, we can drop the the `Oidc` prefix, as it is already implied by the error / the module.

```suggestion
    MissingAdminPasswordKey { secret: ObjectRef<Secret> },
```

_Note: This also needs updating of the context selector below_